### PR TITLE
In the Advanced APIs documentation (section "settings.js"), add a note with a link to https://emsettings.surma.technology

### PR DESCRIPTION
**Pull request by ancientstraits**
Addressing #16563, this pull requests adds a note with this [link](https://emsettings.surma.technology/).

New Image
In this image, the addition is the bottom note.

### CODE REVIEW

File: site/source/docs/api_reference/advanced-apis.rst [hunk 1]
```diff
@@ -38,6 +38,11 @@ not be modified. Note also that the compiler changes some options depending on
 other settings. For example, ``ASSERTIONS`` is enabled by default, but disabled
 in optimized builds (``-O1+``).
 
+.. note::
+  For more information on what options can be used to configure Emscripten, go to
```
[the remark this answers is not in the document]
Comment (kripken) [reply site/source/docs/api_reference/advanced-apis.rst:42]: (seems good to also mention where the source of the settings descriptions is)

[the remark this answers is not in the document]
Comment (ancientstraits) [reply site/source/docs/api_reference/advanced-apis.rst]: That's a great idea, thanks!

**Comment by kripken** [comment]
Hmm, I still don't see CI start to run here...

I was told by CircleCI support last time this happened that it might be because the user has CircleCI set up in their github account but hasn't refreshed it for a long time. @ancientstraits can you try going to https://app.circleci.com/settings/user and clicking "Refresh permissions"?

**Comment by ancientstraits** [comment]
@kripken I refreshed my permissions.

**Comment by ancientstraits** [comment]
Still not working :thinking:

**Comment by kripken** [comment]
Try to push another commit - perhaps it will work only after you did the refresh.

**Comment by ancientstraits** [comment]
@kripken finished the commit

**Comment by ancientstraits** [comment]
come on...

**Comment by ancientstraits** [comment]
What is "archive"?
Anyways, I'm very happy the CircleCI worked.

**Comment by sbc100** [comment]
"archive" is a test for `make dist` which happens to run on github CI .. so requires manual triggering for first time committers.

**Pull request merged**